src/tooltips.tsx

```diff
@@ -294,6 +308,10 @@ export class Tooltips {
 					props.onMouseEnter = (e: Event) => {
 						showFn(e);
 						oldMEFn(e);
+
+						if ((e.target as any).disabled) {
```

Comment:
Should we do that only for buttons? We "can" have the `disabled` attr on many elements. On the other hand, it will probably not hurt anyone to use this approach even for disabled inputs, I guess.
Your call.

Reply:
AFAIK the `mouseleave` event won't be called for any disabled element, so we need to keep this logic here.